pkg/kubelet/oom/oom_watcher_linux_test.go

```diff
@@ -109,15 +109,15 @@ func TestWatcherRecordsEventsForOomEventsCorrectContainerName(t *testing.T) {
 			Pid:                 1000,
 			ProcessName:         "fakeProcess",
 			TimeOfDeath:         time.Now(),
-			ContainerName:       recordEventContainerName,
-			VictimContainerName: "some-container",
+			ContainerName:       recordEventContainerName + "/some-container",
+			VictimContainerName: recordEventContainerName,
 		},
 		{
 			Pid:                 1000,
 			ProcessName:         "fakeProcess",
 			TimeOfDeath:         time.Now(),
-			ContainerName:       "/dont-record-oom-event",
-			VictimContainerName: "some-container",
+			ContainerName:       recordEventContainerName + "/kubepods/some-container",
```

Comment:
Is there a reason we changed this?

Reply:
VictimContainerName should always be the prefix of ContainerName in practice.  I.E.
"Task in /kubepods/poduid/containerid killed as a result of /anothercgroup" isn't possible.